internal/cephes/igam.go

```diff
@@ -89,15 +89,14 @@ import (
  */
 
 const (
-	igamDimK   int     = 25
-	igamDimN   int     = 25
-	maxIter    int     = 2000
-	igam       int     = 1
-	igamC      int     = 0
-	small      float64 = 20
-	large      float64 = 200
-	smallRatio float64 = 0.3
-	largeRatio float64 = 4.5
+	igamDimK       int     = 25
```

Comment:
These all get used in a context where the type is inferable, so the type does not need to be specified here.